curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/613e275749ebd008e74bb62e.md

```diff
@@ -17,7 +17,7 @@ img {
 }
 ```
 
-In this example, `img` elements will have a minimum width of `250px`. And as the viewport grows, the image will grow accordingly to be 25 percent of the viewport width. 
+In the example above, if the viewport is 2000px wide, `25vw` is evaluated to 500px, which is 25% of 2000px. The `max()` function will return `25vw` as the largest value because 500px is larger than `250px`. However, if the viewport is 500px wide, `25vw` is just 125px and is much smaller than `250px`, thus `250px` is the largest value and is the image width.
```

Comment:
```suggestion
In the above example, the width of the image will be 250px if the viewport width is less than 1000 pixels. If the viewport width is greater than 1000 pixels, the width of the image will be 25vw. This is because 25vw is equal to 25% of the viewport width.
```